metadata-ingestion/src/datahub/ingestion/source/tableau_common.py

```diff
@@ -767,7 +767,7 @@ def get_overridden_info(
         ):
             platform_instance = database_hostname_to_platform_instance_map.get(hostname)
 
-    if original_platform in ("athena", "hive", "mysql"):  # Two tier databases
+    if original_platform in ("athena", "hive", "mysql", "teradata"):  # Two tier databases
```

Comment:
We fixed this to get correct automatic lineage to Teradata datasets. Any objections to this?

Reply:
@hsheth2 have you already seen this one? Are you fine with that?

Reply:
yup that one is fine